src/org/usfirst/frc4904/robot/Robot.java

```diff
@@ -71,6 +75,11 @@ public void testInitialize() {}
 	@Override
 	public void testExecute() {}
 
+	byte[] matchConfigurationCANData() {
+		return new byte[] {(byte) DriverStation.getInstance().getAlliance().ordinal(),
+				(byte) DriverStation.getInstance().getLocation(), 0, 0, 0, 0, 0, 0};
```

Comment:
Also, since we have the extra bytes, it might be good to add (autonomous/teleop/disabled) and maybe a flag for near end of match (although that might be too year specific).